.github/workflows/wheels-recipe.yaml

```diff
@@ -145,20 +145,24 @@ jobs:
 
       # Needed to install a specific libomp version later
       - name: Setup Conda
-        uses: conda-incubator/setup-miniconda@d2e6a045a86077fb6cad6f5adf368e9076ddaa8d # v3.1.0
+        uses: conda-incubator/setup-miniconda@505e6394dae86d6a5c7fbb6e3fb8938e3e863830 # v3.1.1
         with:
           python-version: "3.12"
-          channels: conda-forge
+          channels: conda-forge,defaults
```

Comment:
Don't really understand this enough from the pull requests description to approve this. In case you are waiting for it, do you have a pointer to the warning that was raised? E.g. don't see one [here](https://github.com/scikit-image/scikit-image/actions/runs/13578119989/job/37958845956).

Reply:
Please don't mix conda-forge and defaults.
```suggestion
          channels: conda-forge
```

This just isn't necessary and they are growing incompatible.

Whenever I help people troubleshoot this is the first thing I tell them. Please see this little historical note i wrote.
https://conda-forge.org/docs/user/transitioning_from_defaults/#a-historical-note

Reply:
I'm trying to get rid of the warning that the action is currently raising. Can you please suggest a different patch that will achieve the same?

Reply:
just ignore the warning....

Reply:
the transition away from `defaults` has been quite painful, they still maintain `conda` so......  but the general wisdom is "stick to conda-forge" or "defaults", but not together

Reply:
That's really annoying. There's no way around it? Maybe pixi.

Reply:
Lets see what the warning is and how bad it is in the context of the rest of the changes.

Reply:
what job do i look for to see the warning?

Reply:
I think we can just add this option:

```
          conda-remove-defaults: "true"
```

Reply:
I guess that option will eventually be deprecated, but they still have it for now.

Reply:
Re: where it shows: either on the daily cron build, or on branches pushed with name `maintenance/*`.

Reply:
Do you mean this warning? 
![image](https://github.com/user-attachments/assets/549a2351-2220-478d-98ec-c44ecaad5a1a)

Reply:
Yes, but really I mean the white notification box that is added at the top of the run (same message).

Reply:
Lets raise an issue instead. https://github.com/conda-forge/conda-feedstock/issues/262 this isn't worth the churn in scikit-image.